Patches/Core/ThingDefs_Items/Items_Resource_Stuff.xml

```diff
@@ -335,20 +317,6 @@
 		</value>
 	</Operation>
 
-	<Operation Class="PatchOperationReplace">
-		<xpath>Defs/ThingDef[defName="Jade"]/stuffProps/statFactors/MeleeWeapon_CooldownMultiplier</xpath>
-		<value>
-			<MeleeWeapon_CooldownMultiplier>1.15</MeleeWeapon_CooldownMultiplier>
```

Comment:
I'd just leave this--jade is already fairly hopeless as a weapon material (and I don't think you can make much aside from clubs in vanilla), it doesn't need _another_ 15% cooldown.

Moreover, it doesn't make much sense that uranium (almost 6 times denser than jade) has a 110% cooldown, while jade has 130%.